Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM --platform=$TARGETARCH ghcr.io/planetscale/ghcommit:v0.1.74 AS ghcommit
+
+# hadolint ignore=DL3007
+FROM --platform=$TARGETARCH pscale.dev/wolfi-prod/base:latest AS base
```

Comment:
might have to remove it since we cannot pin it and it is fetching from pscale.dev